modules/kvtagger/kvtagger.c

```diff
@@ -378,6 +408,7 @@ kvtagger_parser_new(GlobalConfig *cfg)
   self->super.super.free_fn = kvtagger_parser_free;
   self->super.super.init = kvtagger_parser_init;
   self->scanner = NULL;
+  self->default_selector = NULL;
```

Comment:
We don't explicitly initialize members which are already initialized to the same value by `g_new0()` (`0`).